src/markdown-pages/docs/ecosystem/ecosystem-role-based-access.md

```diff
@@ -0,0 +1,208 @@
+- Not even administrators can change the role of a user nominated as an `owner`.
+- The `owner` role can only be assigned using the Kubernetes property, not via the command-line or REST interface.
+
+## Upgrading to a version of Galasa which supports RBAC
+When upgrading your Galasa service to a version 0.40.0 or later, any existing users will be assigned the role of `admin` without any extra action being required.
+
+Under such circumstances you may wish to assign the role of `tester` to anyone who now should not have administration rights.
+
+Upgrading the existing Galasa service will therefore not require any `owner` to be set up, because your system will already have multiple administrator users with the `admin` role.
+
+
+## Setting up RBAC when you install Galasa into Kubernetes
+
+### How to nominate a user as the Galasa service `owner`
+An "owner" of the Galasa service is a user who has been nominated as the owner of the service, and who has the `owner` RBAC role.
+
+This nomination is made at the time the Galasa service is first installed into the Kubernetes namespace, or 
+when it is subsequently updated using the Galasa Helm chart.
+
+To nominate a user as an `owner` you need to set the property in the Helm values file when the Galasa service is installed or updated.
+
+For example, in the `.values` file used by helm:
+```
+#
+# The list of login ids of the Galasa service owner(s), comma-separated.
+# If a user login id appears in this list, the user will appear to have the 'owner' role, rather than 
+# their normal role.
+#
+# Roles of owner user records cannot be changed. Without changing the configuration at the Kubernetes level.
+# 
+# The users nominated here will always have a role of 'owner'.
+#
+# If the Galasa service owners leave the organisation, or are otherwise unable to administer the system any longer,
+# someone with access to kubernetes can change this value and the user specified will then appear to have the role
+# of owner.
+#
+# Multiple owners can be specified. eg: "user1, user2". All users mentioned will have the 'owner' role until they are 
+# removed from the list
+#
+# If your Galasa service already has a user with the 'admin' role, the `owner` role may not be required.
+galasaOwnersLoginIds: ""
+```
+
+To install the Galasa service, it may be necessary to nominate an `owner` using this mechanism initially, so 
+that when such a user logs-in, they are able to promote other trusted users to the position of `admin`.
+
+After that point, the kubernetes configuration can be changed to remove the `owner` or not.
+
+
+### How to set the default user role
+When a new user initially logs into the Galasa web user interface, a user record is created, and that user is assigned a default user role.
+
+The default user role is configurable by the system installer, who can set a property in the Helm values file when the Galasa service is installed or updated.
+
+For example, in the `.values` file used by helm:
```

Comment:
"values.yaml" instead of ".values" and "Helm" instead of "helm"

Reply:
fixed